packages/minimongo/cursor.js

```diff
@@ -384,23 +383,190 @@ export default class Cursor {
     return handle;
   }
 
+  /**
+   * @summary Watch a query.  Receive callbacks as the result set changes.
+   * @locus Anywhere
+   * @memberOf Mongo.Cursor
+   * @instance
+   * @param {Object} callbacks Functions to call to deliver the result set as it
+   *                           changes
+   * @returns
+   */
+  observeLocal(options) {
+    if (Meteor.isServer)
```

Comment:
It should be possible to use observeLocal for unnamed collections on the server, and when using a ddp client on the server.